resources/sass/app.scss

```diff
@@ -239,86 +237,85 @@ dl.import-meta .download-csv {
 }
 
 .button {
-    padding: $wikit-Button-medium-padding-vertical $wikit-Button-medium-padding-horizontal;
-    font-weight: $wikit-Button-font-weight;
-    line-height: $wikit-Button-line-height;
+    @include ui-text-M($font-weight-bold);
+    padding: $spacing-30 $spacing-75;
     display: inline-block;
-    color: $wikit-Button-normal-neutral-color;
-    border-width: $wikit-Button-border-width;
-    border-style: $wikit-Button-border-style;
-    border-color: $wikit-Button-normal-neutral-border-color;
-    border-radius: $wikit-Button-border-radius;
-    background-color: $wikit-Button-normal-neutral-background-color;
-    transition-duration: $wikit-Button-transition-duration;
-    transition-property: $wikit-Button-transition-property;
+    border-width: $border-width-base;
+    border-style: $border-style-base;
+    border-color: $border-color-base;
+    border-radius: $border-radius-base;
+    outline: none;
```

Comment:
This has no effect, I believe it needs to be included in the focus pseudo-class. will leave a comment there